_config.php

```diff
@@ -1 +1,6 @@
 <?php
+
+namespace SilverStripe\Taxonomy;
+
+
+
```

Comment:
You could just delete this file - it's not needed since we've got things in the `_config` directory